lib/lroPollStrategy.ts

```diff
@@ -315,14 +343,28 @@ function getLocationHeaderValue(response: HttpOperationResponse): string | undef
  * A long-running operation polling strategy that is based on the location header.
  */
 class LocationLROPollStrategy extends LROPollStrategy {
+  private locationStrategyShouldDeserialize(parsedResponse: HttpOperationResponse): boolean {
```

Comment:
very small nit: consider making this module-level for mangling

Reply:
I tried to, but I need access to the initialResponse which is stored in this LROPollStrategy's pollState variable.